examples/getting_started/plot_skore_product_tour.py

```diff
@@ -35,38 +35,13 @@
 # ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 # %%
-# .. note::
-#
-#   If we do not wish for our skore project to be stored in a *temporary* folder, we
-#   can simply create and load the project in the current directory with:
-#
-#   .. code-block:: python
-#
-#     import skore
-#
-#     my_project = skore.open("my_project")
-#
-#   This would create a skore project directory named ``my_project.skore`` in our
+#   Let's start by creating a skore project directory named ``my_project.skore`` in our
 #   current directory.
 
-# %%
-# Here, we start by creating a temporary directory to store our project so that we can
-# easily clean it after executing this example:
-
-# %%
-import tempfile
-from pathlib import Path
-
-temp_dir = tempfile.TemporaryDirectory(prefix="skore_example_")
-temp_dir_path = Path(temp_dir.name)
-
-# %%
-# Then, we create and load the skore project from this temporary directory:
-
 # %%
 import skore
 
-my_project = skore.open(temp_dir_path / "my_project")
+my_project = skore.open("my_project", create=True)
```

Comment:
```suggestion
my_project = skore.open("my_project")
```
`create=True` is the default

Reply:
I like that it's explicit and therefore as a user I feel that I can copy/paste even if I don't have anything before.  
Is it bad practice? or misleading?

Reply:
I don't mind explicit, but the goal of `open` is to do the "right thing" without having to think much about all the arguments. Maybe this is telling us that `open` is a bad name for this function?

Reply:
I think the open name is ok, but i would it more natural to have "create = False" by default.